apps/api/internal/repository/organization.go

```diff
@@ -78,3 +78,46 @@ func (r *OrganizationRepository) CreateOrganization(organization *models.Organiz
 
 	return createdOrganization, nil
 }
+
+func (r *OrganizationRepository) GetOrganization(organization_id string, user_id string) (models.OrganizationResponse, error) {
+	// query := `
+	// 	SELECT
+	// 		id, name, description, created_by
+	// 	FROM
+	// 		organizations
+	// 	WHERE
+	// 		id=$1;
+	// `
+	query := `
+		SELECT
+			o.id,
+			o.name,
+			o.description,
+			o.created_by
+		FROM 
+			organizations o 
+		LEFT JOIN 
+			organization_users ou 
+		ON 
+			o.id = ou.organization_id AND ou.user_id = $2
+		WHERE 
+			o.id = $1 AND (ou.user_id IS NOT NULL OR o.created_by = $2);
+	`
```

Comment:
This query is overkill for this function. We have to separate this query. 

- First we need to check is user member of the organization
   - yes, we need to get the organization
   - no, send response with unauthorized